tests/test_color.py

```diff
@@ -0,0 +1,9 @@
+from oelint_adv import color
+
+
+def test_get_color_by_severity():
+    assert isinstance(color.get_color_by_severity("neutral"), str)
```

Comment:
There are already tests covering that at tests/test_class_integration.py:25+ - please adjust these instead of duplicating code here - thx

Reply:
It would be wise to extract the tests of the `color.py` module and move them to `test_color.py` as they are much easier to spot there apparently. What do you thing?

Reply:
I'm also fine with that, just we shouldn't have duplicate tests in here ;)

Reply:
Had another look there. Actually, I don't think `test_color` is duplicate of what `test_class_integration.py` tests. `test_color` provides the unit tests for `color.py` whereas `test_class_integration.py` tests command line input, thus high-level user interface. I think it make total sense to keep unit tests separate from user input tests. I would just rename `test_class_integration.py` to something more self-explanatory such as `test_command_line_arguments.py`, `test_user_interface.py` or `test_user_input.py`. What do you think @priv-kweihmann?

Reply:
I'm fine with that, if that will come well explained (commit message wise) and preferably as a seperate PR

Reply:
Ok, I'll open another one for that